src/modules/mana/selectors.ts

```diff
@@ -76,3 +85,58 @@ export const getWalletWithdrawals = createSelector<
 >(getWithdrawals, getAddress, (withdrawals, address) =>
   address ? withdrawals.filter((withdrawal) => withdrawal.from === address) : []
 )
+
+export const getTransactionByNetwork = createSelector<
+  RootState,
+  Transaction[],
+  Deposit[],
+  Withdrawal[],
+  Record<Network, AccountTransaction[]>
+>(
+  getTransactionsData,
+  getDeposits,
+  getWithdrawals,
+  (transactions, deposits, withdrawals) => {
+    const result: Record<Network, AccountTransaction[]> = {
+      ETHEREUM: [],
+      MATIC: [],
+    }
+    for (const tx of transactions) {
+      const { network }: { network: Network } = getChainConfiguration(
```

Comment:
instead of `{ network: Network }` let's use the type `ChainConfiguration` from `decentraland-dapps`

Reply:
I didn't make that because the ChainConfiguration is not exported. What do you think is the best solution? keep it like that or exporting ChainConfiguration?

Reply:
ah wait, do we need to type it at all? this is not a generator so shouldn't it be typed already?

Reply:
just `const { network } = getChainConfiguration(...)`